generate/generate.c

```diff
@@ -852,7 +852,7 @@ static int GenerateSource(void)
     CHECK(GenerateCode(CODEGEN_LANGUAGE_CSHARP, "../source/csharp/astronomy.cs", "template/astronomy.cs",  "output"));
     CHECK(GenerateCode(CODEGEN_LANGUAGE_JS, "../source/js/astronomy.ts", "template/astronomy.ts", "output"));
     CHECK(GenerateCode(CODEGEN_LANGUAGE_PYTHON, "../source/python/astronomy/astronomy.py", "template/astronomy.py", "output"));
-    CHECK(GenerateCode(CODEGEN_LANGUAGE_KOTLIN, "../source/kotlin/src/main/kotlin/io/github/cosinekitty/astronomy/Main.kt", "template/astronomy.kt", "output"));
```

Comment:
Kotlin doesn't care much about file name but Java does, Java prefers each file to have one class (or enum) the class to have the same naming as the file name which fortunately it isn't needed in Kotlin but the convention in Kotlin is also follow that optionally which isn't needed to follow here for now 😊